libs/barista-components/table/src/table.ts

```diff
@@ -418,39 +443,29 @@ export class DtTable<T> extends _DtTableBase<T> implements OnDestroy {
   }
 
   /** @internal Exports the filtered display data from the dataSource after being formatted by a displayAccessor. */
-  _exportDisplayData(exportData: T[] = this._filteredData): void {
-    if (this.isEmptyDataSource || typeof exportData[0] != 'object') {
-      return;
+  _exportDisplayData(selectedData?: T[]): void {
+    //nothing to export
+    if (this.isEmptyDataSource) return;
+    //not using DTDataSource, fallback to this._data instead of this._filteredData
+    let exportData: readonly T[];
+    if (selectedData) {
+      exportData = selectedData;
+    } else {
+      exportData =
+        Array.isArray(this._filteredData) &&
+        typeof this._filteredData[0] == 'object'
```

Comment:
We have `isObject` in the core module, you can use this: 
https://github.com/dynatrace-oss/barista/blob/f81416513137d54e281211046a6abc71350ed1db/libs/barista-components/core/src/util/type-util.ts#L54-L56

Reply:
What should the import for this look like? I’ve tried every iteration I can think of, but it doesn’t like importing outside rootDir.

Reply:
Requested changes made, besides removing readonly, which causes a type conflict.